docs/current_docs/manuals/administrator/integrations/index.mdx

```diff
@@ -1,10 +1,10 @@
 ---
-slug: /user-guide/ci
+slug: /manuals/administrator/integrations
```

Comment:
@d3rp3tt3 I'm not sure if we are referring to specific pages from the Dagger Cloud UI. If we are, then please note these URL changes (redirections will be in place though). Inline links within the Dagger Cloud get started guide will be updated separately (not in this PR).